python/ql/test/experimental/dataflow/coverage/test.py

```diff
@@ -77,16 +75,14 @@ def test_imagnumber_literal():
     x = 42j
     SINK(x)  # Flow missing
 
-# 6.2.3. Parenthesized forms
-
 
+# 6.2.3. Parenthesized forms
 def test_parenthesized_form():
-    x = (SOURCE)
+    x = SOURCE
```

Comment:
Did the formatter get a bit aggressive here?

Reply:
Hah, probably. I missed that, thanks!